fineract-provider/src/main/java/org/apache/fineract/infrastructure/codes/CodeConstants.java

```diff
@@ -29,7 +29,7 @@
     public enum CodevalueJSONinputParams {
 
         CODEVALUE_ID("id"), NAME("name"), POSITION("position"), DESCRIPTION("description"), IS_ACTIVE("isActive"), IS_MANDATORY(
-                "isMandatory");
+                "isMandatory"), COUNTRYCODE("28");
```

Comment:
Why did we hardcode the 28 id? This definitely shoud be something like `country` or `countryCode` or something like that.